### PR TITLE
Elector pool support to update-releases-deps

### PR DESCRIPTION
**Pull request by mattcollier**
Depends on one last outstanding PR: https://github.com/digitalbazaar/bedrock-ledger-node/pull/42

### CODE REVIEW

File: lib/config.js [hunk 1]
```diff
@@ -202,12 +202,25 @@ cc('veres-one.config', () => ({
...
-  //
-  // ],
-  // FIXME: Enable validation
+  ledgerConfigurationValidator: [{
+    type: 'VeresOneValidator2017',
+  }, /*{
+    FIXME: validator can't validate sigs using DID keys, how is that supposed
+    to work?  Maintainer DID is generated in ledger.js and the key associated
+    with that is used to sign configuration.  Signature Validator attempts to
+    dereference the key (e.g. did:v1:test:nym:z2cJ...Kx7#authn-key-1) and fails.
```
Comment (dmitrizagidulin) [inline lib/config.js]: Why does it fail? (Like, what's the error etc)

Comment (mattcollier) [reply lib/config.js]: The DIDs being used are not on the ledger yet.  So presumably the public keys cannot be dereferenced.

Comment (mattcollier) [reply lib/config.js]: And they are not on the ledger yet because the ledger does not exist yet, this is the genesis configuration.

Comment (dmitrizagidulin) [reply lib/config.js]: Ahhh right makes sense.

**Pull request merged**